test/runner/requirements/windows-integration.txt

```diff
@@ -2,5 +2,8 @@ cryptography
 jinja2
 junit-xml
 paramiko
-pywinrm
+ntlm-auth
+requests-ntlm
+requests-credssp
+pywinrm[credssp]
```

Comment:
Why do you list the packages above when this would install in one?

Reply:
For the constraints; ansible-test doesn't use `--upgrade` so I wanted to specify minimum versions of those to ensure we'd have message encryption support.

Reply:
But that make sense why you would have both `requests-credssp` and `pywinrm[credssp]` when they are the same thing.

Reply:
Yeah, it's a little duplicated, but that way if we ever add any other requirements to pywinrm[credssp] it'll "just work" by bumping the version (as opposed to explicitly including them here).